config/crd/bases/dynatrace.com_dynakubes.yaml

```diff
@@ -3,7 +3,7 @@ apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
   annotations:
-    controller-gen.kubebuilder.io/version: v0.13.0
+    controller-gen.kubebuilder.io/version: v0.12.1
```

Comment:
update your controller-gen, so we don't flip flop between version annotations 😅

Reply:
Fixed! ty